owner-onboarding-server/src/ov_management/mod.rs

```diff
@@ -0,0 +1,256 @@
+use std::{sync::Arc};
+use openssl::pkey::{Private, PKey};
+use warp::{Filter};
+use std::{usize,str::FromStr};
+use fdo_data_formats::{ownershipvoucher::OwnershipVoucher, types::Guid, DeserializableMany};
+use serde::{Serialize, Deserialize};
+use warp::{hyper::StatusCode, http::HeaderValue, Buf};
+use crate::{OwnerServiceUD};
+use thiserror::Error;
+use serde_json::{json};
+
+
+pub(crate) fn ov_filter(session_store:Arc<OwnerServiceUD>) 
+-> impl Filter<Extract = impl warp::Reply, Error = warp::Rejection> + Clone
+{       
+        let store1 = session_store.clone();
```

Comment:
on dropping this and adding session_store.store to both of the filters,

getting error: move occurs because `session_store` has type `std::sync::Arc<OwnerServiceUD>`, which does not implement the `Copy` trait